src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -589,6 +578,9 @@ void Frustum::pup(PUP::er& p) {
     // data from an old version.
     inner_radius_ = std::numeric_limits<double>::signaling_NaN();
   }
+  if (version >= 3) {
+    p | zeta_distribution_value_;
```

Comment:
Add an `else` clause which runs when unpacking data written by previous version of the code. It should set `zeta_distribution_value_` to the value use before this change.